tests/slinky/slinky_test.go

```diff
@@ -0,0 +1,101 @@
+package slinky_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/types/module/testutil"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/bank"
+	"github.com/cosmos/cosmos-sdk/x/gov"
+	"github.com/strangelove-ventures/interchaintest/v8"
+	"github.com/strangelove-ventures/interchaintest/v8/chain/cosmos"
+	"github.com/strangelove-ventures/interchaintest/v8/ibc"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/skip-mev/slinky/tests/integration"
+	marketmapmodule "github.com/skip-mev/slinky/x/marketmap"
+	"github.com/skip-mev/slinky/x/oracle"
+	oracletypes "github.com/skip-mev/slinky/x/oracle/types"
+)
+
+var (
+	dockerImage = ibc.DockerImage{
+		// NOTE: "Repository" field is the name of the image tag generated in the tests/justfile job "test-slinky".
+		Repository: "warden/warden-protocol",
+		Version:    "latest",
```

Comment:
this is what you will have to update if you need to change the dockerfile tag name in the `just test-slinky` command.